src/main/java/de/retest/web/selenium/TestHealer.java

```diff
@@ -169,18 +169,12 @@ private WebElement findElementByCssSelector( final ByCssSelector by ) {
 
 	private String retrieveUsableCssSelector( final ByCssSelector by ) {
 		final String rawSelector = ByWhisperer.retrieveCssSelector( by );
-		if ( rawSelector.startsWith( "#" ) ) {
-			throw new IllegalArgumentException(
-					"To search for element by ID, use `By.id()` instead of `#id` as CSS selector." );
-		}
-		if ( !rawSelector.startsWith( "." ) ) {
-			throw new IllegalArgumentException(
-					"To search for element by tag, use `By.tagName()` instead of `tag` as CSS selector." );
-		}
 		// remove leading .
 		final String selector = rawSelector.substring( 1 );
```

Comment:
If raw selector doesn't have a leading '.' it might be removing the wrong thing.